gdk-config.json

```diff
@@ -0,0 +1,16 @@
+{
+  "component" :{
+    "aws.greengrass.LogManager": {
+      "author": "AWS",
+      "version": "NEXT_PATCH",
+      "build": {
+        "build_system": "maven"
+      },
+      "publish": {
+        "bucket": "gg-dev-artifacts",
+        "region": "us-west-2"
```

Comment:
Placeholder values

Reply:
We have the same value for cda it wont reveal anything public